### PR TITLE
Update Open Terms Archive documents fetch URL

### PR DESCRIPTION
**Pull request by clementbiron**

### CODE REVIEW

File: declarations/Open Terms Archive.history.json [hunk 1]
```diff
@@ -4,13 +4,23 @@
...
   "Imprint": [
     {
       "fetch": "https://opentermsarchive.org/legal-notice",
       "select": ".TextContent_textContent__ToW2S",
       "validUntil": "2024-03-18T18:30:09Z"
+    },
+    {
+      "fetch": "https://opentermsarchive.org/legal-notice",
+      "select": ".textcontent",
+      "validUntil": "2024-04-16T18:30:11.000Z"
```
Comment (MattiSG) [inline declarations/Open Terms Archive.history.json:23]: Why do the `validUntil` dates differ for both documents when the change is a consequence of https://github.com/OpenTermsArchive/opentermsarchive.org/pull/318?

**Review by MattiSG — DISMISSED** [review]

File: declarations/Open Terms Archive.history.json (hunk 1 above)
Comment (clementbiron) [reply declarations/Open Terms Archive.history.json:23]: It's a mistake, I've corrected it.

Comment (MattiSG) [reply declarations/Open Terms Archive.history.json:23]: https://github.com/OpenTermsArchive/opentermsarchive.org/pull/318 was merged on April 19th. How can the date of the switch here be April 16th? 🤔

**Comment by clementbiron** [comment]
Seen in sync with @Ndpnt

**Review by Ndpnt — APPROVED** [review]

**Pull request merged**